pynest/nest/lib/hl_api_info.py

```diff
@@ -55,23 +58,23 @@ def authors():
 
 
 @check_stack
-def helpdesk(browser="firefox"):
-    """Open the NEST helpdesk in the given browser.
+def helpdesk():
+    """Open the NEST helpdesk in browser.
 
-    The default browser is firefox.
-
-    Parameters
-    ----------
-    browser : str, optional
-        Name of the browser to use
+    The system default browser.
+    /helpdesk << /command (firefox) >> SetOptions in ~/.nestrc is now obsolete
```

Comment:
I would drop this line of documentation, since it will confuse most users. The nestrc line is still relevant for the SLI level, so I'd leave it in nestrc.